### PR TITLE
Introducing Vespa CLI

### PR DESCRIPTION
**Pull request by mpolden**
@bratseth

### CODE REVIEW

File: _posts/2021-09-23-introducing-vespa-cli.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- Feed and query documents
+- Send custom requests with automatic authentication
+
+To install Vespa CLI, choose one of the following methods:
+
+- Homebrew: `brew install vespa-cli`
+- [Download from GitHub](https://github.com/vespa-engine/vespa/releases)
+
+To learn how to use Vespa CLI check out our getting started guides:
+- [Open source Vespa](https://docs.vespa.ai/en/getting-started.html)
```
Comment (mpolden) [inline _posts/2021-09-23-introducing-vespa-cli.md:45]: This should link to `vespa-cli.html` when it's available.

File: _posts/2021-09-23-introducing-vespa-cli.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+repository](https://github.com/vespa-engine/vespa/tree/master/client/go). If you
+encounter problems or want to provide feedback on Vespa CLI, feel free to [file
+a GitHub issue](https://github.com/vespa-engine/vespa/issues/new/choose).
+
+## In action
+
+This screencast demonstrates Vespa CLI being used with [Vespa
+Cloud](https://cloud.vespa.ai).
+
+<script id="asciicast-UyZQXh1TxLo43ON0CMqgFxEj0" src="https://asciinema.org/a/UyZQXh1TxLo43ON0CMqgFxEj0.js" async></script>
```
Comment (bratseth) [inline _posts/2021-09-23-introducing-vespa-cli.md:59]: How about just putting this on top, with no commentary?

**Review by bratseth — APPROVED** [review]
Great!

File: _posts/2021-09-23-introducing-vespa-cli.md (hunk 2 above)
Comment (mpolden) [reply _posts/2021-09-23-introducing-vespa-cli.md:59]: I think I prefer it this way. There's already an image (screenshot) at the top.

If I replace the image with the video, there won't be any image shown on the front page.

Comment (bratseth) [reply _posts/2021-09-23-introducing-vespa-cli.md:59]: Okay, but you could get around that, e.g by displaying the image in a hidden div tag?

**Review by kkraune — APPROVED** [review]
Great!

**Review by bratseth — APPROVED** [review]

**Pull request merged**